src/store/user-store.js

```diff
@@ -1,16 +1,48 @@
-import { makeAutoObservable } from "mobx";
+import { makeAutoObservable, flow } from "mobx";
 
 class UserStore {
   isLoggedIn = false;
 
+  userResponse = {};
+
   constructor(rootStore) {
     this.rootStore = rootStore;
-    makeAutoObservable(this);
+    makeAutoObservable(this, {
+      fetchUserInformation: flow
+    });
   }
 
   setIsLoggedIn(value) {
     this.isLoggedIn = value;
   }
+
+  setUserResponseData(data) {
+    this.userResponse = data;
+  }
```

Comment:
Ignore this, it was done in [the previous lesson](https://github.com/kenttk/marks-side-project/pull/26)